### ISSUE TITLE
Cancel a download 

### ISSUE THREAD

**Issue by TheFrustratedDeveloper** [FaisalUmair/udemy-downloader-gui#135]
Provide a button to cancel a download rightaway

**Comment by FaisalUmair** [comment]
By cancel do you mean to it should delete all the downloaded files? Because there is already a pause button which pauses download and keeps the downloaded files.

**Comment by TheFrustratedDeveloper** [comment]
yes, it should delete all the files if i press cancel the download

**Comment by joseJavierOjeda** [comment]
It would be nice to have a button to cancel and delete all files from a download

**Comment by erickv94** [comment]
I think the same, it would be nice to have a button to cancel, because Downloads List is full

**Comment by Ryan-ED** [comment]
Any updates on this?
